hugo/layouts/index.html

```diff
@@ -28,8 +28,7 @@
     {{ end }}
 
     {{ partial "footer/slack.html" . }}
```

Comment:
It will be used in the project page, so it should be in the footer configurable template